recipes/targetscan/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "targetscan" %}
+{% set version = "7.0" %}
+{% set sha256 = "6e23c72a920f2a58ecf783fe33a6929bc16bbeb5359b020505c0fc95019bd749" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://www.targetscan.org/vert_80/vert_80_data_download/targetscan_70.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+    - perl
+  run:
+    - perl
+
+test:
+  commands:
+    - perl --version
+    - targetscan_70.pl > /dev/null
+
+about:
+  home: https://www.targetscan.org/vert_80/
+  license: MIT
+  summary: Search for predicted microRNA targets in mammals
+
+extra:
+  skip-lints:
+    - license_file
```

Comment:
The license is not really optional. There needs to be a license in place that allows redistribution- see the checklist at https://bioconda.github.io/contributor/guidelines.html#bioconda-recipe-checklist.